### DISCUSSION TITLE
useFieldArray: UseFieldArrayProps - Compatibility with remix-hook-form

### DISCUSSION THREAD

**Discussion by drucku** [Code-Forge-Net/remix-hook-form#108] [General]
I’m working on a project using useFieldArray from react-hook-form for dynamic form handling and came across remix-hook-form. I wanted to clarify if remix-hook-form fully supports the useFieldArray hook from react-hook-form, as it doesn’t seem to be working as expected in my setup.

The official react-hook-form documentation mentions that useFieldArray is used for managing dynamic field arrays, ensuring better user experience and performance. However, when I try to use it within remix-hook-form, I’m encountering issues where the form state doesn’t update as expected.

Can you confirm if useFieldArray is fully supported in remix-hook-form? If it is, could you provide any additional steps or considerations that I may have missed during setup? If not, are there any workarounds or alternative approaches to achieve dynamic form functionality?

```
const { fields, append, remove } = useFieldArray({
  control,
  name: "fieldArrayName",
});
```

**Comment by AlemTuzlak** [comment]
Where does your control come from?

**Reply by drucku** [reply]
I import it like that:  const { control, trigger } = useRemixFormContext()

I know noticed, when I use regular inputs it works. The problem is my mantine text input implementation.

**Comment by drucku** [comment]
It works
